### ISSUE TITLE
[Feature] Add activity-indicator

### ISSUE THREAD

**Issue by Glitchy-Tozier** [thetwom/Tuner#13]
Currently, when the tone gets too quiet, the line just stops moving.

With instruments that don't have much fluctuation within a single tone, the user has no idea whether the recording actually stopped or whether the note still is displayed, because all the user sees is this:
![20210629_222220.jpg](https://user-images.githubusercontent.com/59611881/123862317-7c2a5380-d928-11eb-906a-24a141a177a8.jpg)

I have to suggestions on how to fix this
1. Add some lines that trail across the screen when the dot "is moving towards the right"
2. Make it so that the trail always continues, but the trail changes color when the signal is insufficient.

Either fix is fine. Of course, I'm open to suggestions  :)

**Comment by thetwom** [comment]
You are right, I will definitely think of improving this when I find time.

**Comment by thetwom** [comment]
I am currently trying around how to detect if there is a valid signal or not. Unfortunately, this is not straight forward, since the loudness of the input signal is very device dependent ... so if you are willing, you could try a test version. If you install from google play, I introduced an open  test: [open test](https://play.google.com/store/apps/details?id=de.moekadu.tuner)
Otherwise I can provide an apk here on github ..

**Comment by thetwom** [comment]
Activity indication is now published in version [v.1.3.0](https://github.com/thetwom/Tuner/releases/tag/v1.3.0).

**Comment by Glitchy-Tozier** [comment]
Looks pretty good! 

If you wanted, you could make the inactive line even brighter and give the dot another colour, but that's nitpicking. Overall, really nice!

**Comment by thetwom** [comment]
not sure if this is a matter of taste. Any reason why you would make the line brighter and the dot another color?

**Comment by Glitchy-Tozier** [comment]
To make the two states (active, inactive) more visually distinct.

Curremtly it's okay, but I feel like I have to squint a little, focus my attention, to see what state we're in. Other tuners are designed to male this obvious even when just taking a quick glance.

Maybe making the line brighter isn't the beat choice, but giving it a distinct color(red?) might make the information more accessible.
